src/corehost/cli/coreclr.cpp

```diff
@@ -52,6 +62,7 @@ bool coreclr::bind(const pal::string_t& libcoreclr_path)
     coreclr_initialize = (coreclr_initialize_fn)pal::get_symbol(g_coreclr, "coreclr_initialize");
     coreclr_shutdown = (coreclr_shutdown_fn)pal::get_symbol(g_coreclr, "coreclr_shutdown_2");
     coreclr_execute_assembly = (coreclr_execute_assembly_fn)pal::get_symbol(g_coreclr, "coreclr_execute_assembly");
+    coreclr_create_delegate = (coreclr_create_delegate_fn)pal::get_symbol(g_coreclr, "coreclr_create_delegate");
```

Comment:
I would suggest doing the following instead:

> `coreclr_create_delegate = (declspec(coreclr_create_delegate))pal::get_symbol(g_coreclr, "coreclr_create_delegate");`

The `declspec()` keyword is rather helpful and makes the intent here clearer.

Reply:
Removed this function since the new implementation doesn't use it.